regression/contracts/assigns_enforce_15/main.c

```diff
@@ -20,10 +20,16 @@ int baz() __CPROVER_ensures(__CPROVER_return_value == global)
   return global;
 }
 
+void qux(void) __CPROVER_assigns()
```

Comment:
The semantics of an empty assigns clause should be that the function doesn't change any (global) arguments, right?

Reply:
Yes!